pandas/tests/io/formats/test_to_html.py

```diff
@@ -477,3 +477,28 @@ def test_to_html_float_format_no_fixed_width(self, datapath):
         df = DataFrame({'x': [100.0]})
         expected = expected_html(datapath, 'gh22270_expected_output')
         assert df.to_html(float_format='%.0f') == expected
+
+    @pytest.mark.parametrize("render_links, file_name", [
+        (True, 'render_links_true'),
+        (False, 'render_links_false'),
+    ])
+    def test_to_html_render_links(self, render_links, file_name, datapath):
+        # GH 2679
+        data = [
+            {
+                'foo': 0,
+                'bar': 'http://pandas.pydata.org/?q1=a&q2=b',
+                None: 'pydata.org',
+            },
+            {
+                'foo': 0,
+                'bar': 'www.pydata.org',
+                None: 'pydata.org',
+            },
+        ]
+        df = DataFrame(data, columns=['foo', 'bar', None],
```

Comment:
Also I think you can improve the construction here. Would be easier I think if you just pass a list of lists for the values. You are explicitly setting `columns` so the list of dicts is superfluous

Reply:
Fixed in 7b5b323